src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.PublishVerticalManifest/Microsoft.DotNet.SourceBuild.Tasks.PublishVerticalManifest.csproj

```
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+  </PropertyGroup>
```

Please enable nullable for this project so that we get nullable reference type diagnostics and avoid potential NREs.

```suggestion
    <Nullable>enable</Nullable>
  </PropertyGroup>
```

---

Enabled. Not sure if msbuild will work with the `required` and `init` keywords I had to add to parameters tho, we'll see